oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/pipelined/TransformStageStatistics.java

```diff
@@ -72,10 +71,6 @@ public BoundedHistogram getSplitDocumentsHistogram() {
         return splitDocumentsHistogram;
     }
 
-    public BoundedHistogram getEmptyNodeStateHistogram() {
```

Comment:
Won't this help in figuring out garbage in mongo. If yes, i think we should keep it.

Reply:
We still report the total number of garbage documents and the percentage. And Oak now has FullGC to clear this garbage, so the presence of garbage should become less of a problem.